js/test/threepio.test.js

```diff
@@ -1,23 +1,96 @@
+import * as tf from '@tensorflow/tfjs';
 import Threepio from '../src/threepio';
-import { add } from './fixtures/pytorch';
+import {
+  add,
+  matmul,
+  mean,
+  mul,
+  div,
+  eq,
+  sum,
+  argmax,
+  t,
+  softmax,
+  relu
+} from './fixtures/pytorch';
 const { test, describe, expect } = global; // import jest from global
 
 describe('Threepio', () => {
-  const brokenAdd = {
-    ...add,
-    kwargs: undefined
+  const threepio = new Threepio('torch', 'tfjs');
+
+  const checkAnswer = (result, answer) => {
+    const eq = tf
+      .equal(result, answer)
+      .all()
+      .dataSync();
+    expect(eq).toEqual(new Uint8Array([1]));
+  };
+
+  const processTests = command => {
+    for (const [i, input] of command.inputs.entries()) {
+      const translation = threepio.translate(input);
+      const result = translation.executeRoutine();
+      const answer = command.answers[i];
+      checkAnswer(result, answer);
+    }
   };
-  const threepio = new Threepio({});
 
   test('returns itself', () => {
     expect(threepio).toBeInstanceOf(Threepio);
   });
 
-  test('translate basic function', () => {
-    threepio.translate('tfjs', add);
+  test('translates add', () => {
+    processTests(add);
+  });
+
+  test('translates matmul', () => {
+    processTests(matmul);
+  });
+
+  test('translates mean', () => {
+    processTests(mean);
+  });
+
+  test('translates mul', () => {
+    processTests(mul);
+  });
+
+  test('translates div', () => {
+    processTests(div);
+  });
+
+  test('translates eq', () => {
+    processTests(eq);
+  });
+
+  test('translates sum', () => {
+    processTests(sum);
+  });
+
+  test('translates argmax', () => {
+    processTests(argmax);
+  });
+
+  // TODO: Write tests for kwargs use of dim
+
+  test('translates transpose', () => {
+    processTests(t);
+  });
+
+  test('translates softmax', () => {
```

Comment:
softmax requires a special test as we need to use `toBeCloseTo` due to rounding errors